EIPS/eip-2294.md

```diff
@@ -0,0 +1,41 @@
+---
+eip: 2294
+title: Explicit bound to Chain ID size
+description: Adds a maximum value to the Chain ID parameter to avoid potential encoding issues that may occur when using large values of the parameter.
+author: Zainan Victor Zhou (@xinbenlv), Alex Beregszaszi (@axic)
+discussions-to: https://ethereum-magicians.org/t/eip-2294-explicit-bound-to-chain-id/11090
+status: Draft
```

Comment:
```suggestion
status: Withdrawn
```
Either must be number 5728 or must be published as withdrawn

Reply:
As an author, I am fine ether way. 

But as a champion of an existing EIP, I personally prefer to keep the number 2294. This is because this is a EIP that the original lead author agree to let someone (me) help take over and champion. I am not starting a new EIP to compete with this existing EIP. I think for the credit to the prior work performed by authors of this EIP.

@Pandapip1 does this argument make sense?

Reply:
@Pandapip1 and I discussed on discord, and we're okay assigning the original EIP number (2294) as long as one of the original authors (@fubuloubu, @axic) approves this PR.

Reply:
Can confirm.